docs/operator/notifications.md

```diff
@@ -0,0 +1,29 @@
+| When the replication rule transitioning into OK_STATE for DIDs which are closed            | RULE_OK         | if not did['open']: payload = {'scope': rule.scope.external, 'name': rule.name, 'rule_id': rule.id} if rule.scope.vo != 'def': payload['vo'] = rule.scope.vo                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| For replication replication rule, for each 10% of progress                                 | RULE_PROGRESS   | if rule.notification == RuleNotification.PROGRESS: payload = {'scope': rule.scope.external, 'name': rule.name, 'rule_id': rule.id, 'progress': \_\_progress_class(rule.locks_replicating_cnt, total_locks)} if rule.scope.vo != 'def': payload['vo'] = rule.scope.vo                                                                                                                                                                                                                                                                                                                                                                   |
+| When the replication rule transitioning into OK_STATE for each DATASET covered by the rule | DATASETLOCK_OK  | for dataset_lock in dataset_locks: payload = {'scope': dataset_lock.scope.external, 'name': dataset_lock.name, 'rse': get_rse_name(rse_id=dataset_lock.rse_id, session=session), 'rse_id': dataset_lock.rse_id, 'rule_id': rule.id} if dataset_lock.scope.vo != 'def': payload['vo'] = dataset_lock.scope.vo                                                                                                                                                                                                                                                                                                                           |
+| When the transfer is done                                                                  | transfer-done   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
Instead of the `transfer_status` one from above, we should just fill these. 
transfer-done is when a transfer has been completed etc. 
payload should look similar, but I believe there should be more timestamps

Reply:
Yes, the structure of the payload is pretty long sequence of if's (https://github.com/esilvaju/rucio/blob/6eac5dcc30cb6dac427cbc5070d46c170ae7fedf/lib/rucio/daemons/hermes/hermes.py#L230-L311) this is why I did not added the code here.

Looking at your examples I believe will be something like this:

```{'src-rse': 'TESTRSE', 'dst-rse': 'TESTRSE',payload["activity"], 'bytes': 1000000, 'transferred_at' : '2023-07-05 14:51:55.378549'}```